Fix easy missing moves in function_schema_parser

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #151850
* #151849
* #151810
* #151807
* #151806
* __->__ #151805
* #151804
* #151803
* #151802
* #151801

Just some straightforward not-moving-upon-return.

Differential Revision: [D73376718](https://our.internmc.facebook.com/intern/diff/D73376718/)

cc @EikanWang @jgong5 @wenzhe-nrv @sanchitintel

swolchok committed:
Fix easy missing moves in function_schema_parser

Just some straightforward not-moving-upon-return.

Differential Revision: [D73376718](https://our.internmc.facebook.com/intern/diff/D73376718/)

[ghstack-poisoned]

swolchok committed:
Update on "Fix easy missing moves in function_schema_parser"


Just some straightforward not-moving-upon-return.

Differential Revision: [D73376718](https://our.internmc.facebook.com/intern/diff/D73376718/)

cc EikanWang jgong5 wenzhe-nrv sanchitintel

[ghstack-poisoned]

swolchok committed:
Update on "Fix easy missing moves in function_schema_parser"


Just some straightforward not-moving-upon-return.

Differential Revision: [D73376718](https://our.internmc.facebook.com/intern/diff/D73376718/)

cc EikanWang jgong5 wenzhe-nrv sanchitintel

[ghstack-poisoned]

malfet reviewed (approved):
Sure, though wouldn't compiler be able to figure it out on its own?

swolchok committed:
Update on "Fix easy missing moves in function_schema_parser"


Just some straightforward not-moving-upon-return.

Differential Revision: [D73376718](https://our.internmc.facebook.com/intern/diff/D73376718/)

cc EikanWang jgong5 wenzhe-nrv sanchitintel

[ghstack-poisoned]

cyyever commented:
@pytorchbot merge -i

swolchok committed:
rebase and minor fixes as requested on "Fix easy missing moves in function_schema_parser"


Just some straightforward not-moving-upon-return.

Differential Revision: [D73376718](https://our.internmc.facebook.com/intern/diff/D73376718/)

cc EikanWang jgong5 wenzhe-nrv sanchitintel

[ghstack-poisoned]

swolchok committed:
Update on "Fix easy missing moves in function_schema_parser"


Just some straightforward not-moving-upon-return.

Differential Revision: [D73376718](https://our.internmc.facebook.com/intern/diff/D73376718/)

cc EikanWang jgong5 wenzhe-nrv sanchitintel

[ghstack-poisoned]
